de.hamstersimulator.objectsfirst.inspector/src/main/java/de/hamstersimulator/objectsfirst/inspector/model/ClassFactory.java

```diff
@@ -147,7 +154,14 @@ private MethodViewModel createConstructorViewModel(final Constructor<?> construc
         final Function<List<?>, ?> construct = params -> {
             try {
                 return constructor.newInstance(params.toArray());
-            } catch (final IllegalAccessException | InvocationTargetException | InstantiationException e) {
+            } catch (final InvocationTargetException targetEx) {
+                final Throwable cause = targetEx.getCause();
```

Comment:
This code is a duplicate of the one above, isn't it. Helper method?

Reply:
extracted with e6134bcd7fbb0464e2ff178c82a4a253b13600b7